src/components/Home.js

```diff
@@ -1,10 +1,11 @@
 import React, { Component } from 'react';
-import { Container, Row } from 'reactstrap';
+import { Container, Row, Col } from 'reactstrap';
 
 import '../styles/Home.css';
 import DisplayComponent from './DisplayComponent';
 import { connect } from 'react-redux';
 import LeftSideBar from './LeftSideBar';
+import YearSelection from './YearSelection.js';
```

Comment:
Can you clean up these import statements a bit? I'd prefer not having the file extension at all. Also, separate library imports with css and our own component imports? I know it's not your change, but since you touched it..

Reply:
Changed to YearSelector